corehq/apps/domain/migrations/0014_appreleasemodesetting_squashed_0015_auto_20220930_2012.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-10-11 08:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('domain', '0014_appreleasemodesetting'), ('domain', '0015_auto_20220930_2012')]
```

Comment:
was `0014_appreleasemodesetting` added by this PR?

Reply:
Yes

Reply:
But its not showing in the PR?
If its meant to replace the migrations, they would still need to be present so that django does not complain about missing migrations.
If you were just merging them to make a single migration for the PR, you don't need to add replaces here.

Reply:
Replaced with a new migration.

Reply:
thank you for fixing this @akashkj

Reply:
what was needed for this to be done on staging?

Reply:
No change done on staging.